subliminal/converters/subscene.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+"""
+"""
+from babelfish import Language, LanguageReverseConverter
+
+from ..exceptions import ConfigurationError
+
+
+# alpha3 extracted from `https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes`
+# Subscene language list extracted from it's upload form
+from_subscene = {
+        'Farsi/Persian': 'fas', 'Greek': 'ell', 'Greenlandic': 'kal',
+        'Malay': 'msa', 'Pashto': 'pus', 'Punjabi': 'pan', 'Swahili': 'swa'
+}
+
+to_subscene = {val: key for key, val in from_subscene.items()}
+
+exact_languages_alpha3 = [
+        'ara', 'aze', 'bel', 'ben', 'bos', 'bul', 'cat', 'ces', 'dan', 'deu',
+        'eng', 'epo', 'est', 'eus', 'fin', 'fra', 'heb', 'hin', 'hrv', 'hun',
+        'hye', 'ind', 'isl', 'ita', 'jpn', 'kat', 'kor', 'kur', 'lav', 'lit',
+        'mal', 'mkd', 'mni', 'mon', 'mya', 'nld', 'nor', 'pol', 'por', 'ron',
+        'rus', 'sin', 'slk', 'slv', 'som', 'spa', 'sqi', 'srp', 'sun', 'swe',
+        'tam', 'tel', 'tgl', 'tha', 'tur', 'ukr', 'urd', 'vie', 'yor'
+]
+
+# TODO: specify codes for unspecified_languages
+unspecified_languages = [
+        'Big 5 code', 'Brazillian Portuguese', 'Bulgarian/ English',
```

Comment:
What's needed here?

Reply:
These are the languages that I couldn't find the `babelfish` equivalent and are added just to have a list of all possible languages supported by Subscene and handle them in case they appear in the subtitle list.

Reply:
Brazillian Portugues: Language('por', 'BR')

Reply:
Added.